lib/server.js

```diff
@@ -263,6 +266,7 @@ class NodeServer extends ServerBase {
   constructor (client, opts) {
     super(client, opts)
 
+    this.nat = opts.nat || false
```

Comment:
honestly, we dont need nat in the http server

Reply:
Updated to remove NAT for the HTTP server.